TipCatDotNet.Api/Services/HospitalityFacilities/MemberService.cs

```diff
@@ -85,6 +87,28 @@ async Task<Result<int>> CreateAndRegisterInvitation(int memberId)
         }
 
 
+        public Task<Result<MemberResponse>> TransferToFacility(MemberContext memberContext, int facilityId, int memberId, int accountId,
+            CancellationToken cancellationToken = default)
+        {
+            return Result.Success()
+                .EnsureCurrentMemberBelongsToAccount(memberContext.AccountId, accountId)
+                .BindWithTransaction(_context,
```

Comment:
Probably we need no transactions here, because the transfer is a single change here. Other methods just read.